app/pubspec.yaml

```diff
@@ -149,6 +149,7 @@ dependencies:
   phosphor_flutter: ^2.1.0
   device_calendar: ^4.3.3-20240801
   quickalert: ^1.1.0
+  extension_nullable: ^1.0.1
```

Comment:
No, I'd not suggest using it. The plugin itself isn't mature and doesn't provide enough documentation and source code.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2179/commits/a80e89005edc85169d05132c6974faf66f8e55e7